src/core/services/OptionsService.ts

```diff
@@ -126,6 +126,7 @@ class OptionsService {
       await axios.post("http://app:4000/reload");
       await axios.post("http://api_app:4000/reload");
       await axios.post("http://webhook_app:4000/reload");
+      await axios.post("http://telegram_bot:4000/reload");
```

Comment:
This needs to either be enabled conditionally with an environment variable or the possible exception needs to be ignored as this will fail for clients without the Telegram bot

Reply:
Should now be solved, do we want to do it like this? For #342 I would move the post methods to the `NetworkCommunicatorService`

Reply:
I think it needs to handle any failure for the core apps (app, api_app, webhook_app), rather than just 404s. They are very unlikely to return 404s, the more likely problems are network failure/timeout.

To be honest I'd just wrap the telegram_bot in an extra try catch for now so it ignores any failure.
```ts
try {
  // TODO: remove hardcoded service references
  await axios.post("http://app:4000/reload");
  await axios.post("http://api_app:4000/reload");
  await axios.post("http://webhook_app:4000/reload");
  try {
   await axios.post("http://telegram_bot:4000/reload");
  } catch (err) {}
} catch (error) {
  log.error("Failed to notify webhook of options change", error);
}
```

Reply:
@wpf500 I have merged #342 into this PR, this makes it easier to implement your request changes. I have now defined the services in `NetworkCommunicatorService.services` and made the telegram bot optional, network errors from optional services are ignored, what do you think?